frontend/test/metabase/scenarios/visualizations/drillthroughs/dash_drill.cy.spec.js

```diff
@@ -215,9 +215,7 @@ describe("scenarios > visualizations > drillthroughs > dash_drill", () => {
 
             cy.wait("@dataset");
             cy.findByText("Category is Doohickey");
-            cy.get(".Visualization .ReactVirtualized__Grid").within(() => {
-              cy.findByText("177"); // Doohickeys for 2016
-            });
+            cy.findByText("177"); // Doohickeys for 2016
```

Comment:
The `within` was causing a failure when "177" was on the page. I assume we're re-rendering a new Grid with the content after Cypress had already selected it. It seemed fine to drop the `within` anyways.

Reply:
@nemanjaglumac I'm sure I have seen other tests doing this, so we should probably change those.

Reply:
If and whenever I notice a problem with this pattern, I'll definitely change it.

Reply:
It seems like `within` on a `Visualization` typically works. I'm not sure why this was different.

I saw a similar failure the other day with the tooltip and action menu popovers on the region map. `popover().within` failed because it had grabbed the wrong popover and it didn't retry that selection when `within` failed. For cases like that, it could be nice to have a helper that let's us scope a query like `within` but retries like `and`/`should`.